tensorflow/core/kernels/training_ops.cc

```diff
@@ -2973,8 +2957,7 @@ namespace functor {
 DECLARE_GPU_SPEC(Eigen::half);
 DECLARE_GPU_SPEC(float);
 DECLARE_GPU_SPEC(double);
-#if !defined(TENSORFLOW_USE_NVCC) && \
-    !defined(TENSORFLOW_USE_ROCM)  // TODO(b/143684500): Eigen to support
+#if !defined(TENSORFLOW_USE_NVCC)  && !defined(TENSORFLOW_USE_ROCM)  // TODO(b/143684500): Eigen to support
```

Comment:
you will need to `clang-format` this line and other like it, else upstream sanity checks will fail when you file the PR